### PR TITLE
CD-187 components

### PR DESCRIPTION
**Pull request by left23**
1. Install components module
2. Set theme to cd-187-components branch
3. Set frontpage to /node so we can override with page--front template and print components

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -102,10 +102,11 @@
         "drupal-composer/drupal-scaffold": "^2.6",
         "drupal-composer/preserve-paths": "^0.1.5",
         "drupal/admin_denied": "^1.0",
+        "drupal/components": "^1.1",
```
Comment (left23) [inline composer.json]: @rupl This needs `composer install` . If there are issues (maybe caused by the theme branch below), delete the .lock file first.

File: config/core.extension.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ module:
   big_pipe: 0
   block: 0
   breakpoint: 0
+  components: 0
```
Comment (left23) [inline config/core.extension.yml]: @rupl This needs a config import to enable the Components module
`drush cex`
https://drushreference.com/drush-8x/config/config-export/

Comment (rupl) [reply config/core.extension.yml]: wouldn't I be doing a `drush cim` to import the config in this PR?

Comment (left23) [reply config/core.extension.yml]: Oops, yes, import

**Pull request merged**